lib/cache.js

```diff
@@ -0,0 +1,68 @@
+const fs = require('fs-extra');
+const path = require('path');
+const co = require('co');
+
+const cacheFilePath = path.join(__dirname, '__cache.json');
```

Comment:
hmm, we should not store cache in the package directory. simply use something like https://github.com/sindresorhus/conf